okhttp/src/jvmMain/kotlin/okhttp3/Response.kt

```diff
@@ -298,6 +303,8 @@ actual class Response internal constructor(
 
     actual open fun priorResponse(priorResponse: Response?) = commonPriorResponse(priorResponse)
 
+    actual open fun trailers(trailersFn: (() -> Headers)): Builder = commonTrailers(trailersFn)
```

Comment:
This allows creating a Response for a bridging interceptor, without an Exchange.

Reply:
Nice.